src/main/kotlin/me/zeroeightsix/kami/target/CategorisedTargets.kt

```diff
@@ -119,6 +130,36 @@ enum class BlockCategory(override val belongsFunc: (Block) -> Boolean) : Categor
     override val provider: ResettableLazy<List<Block>?> = ResettableLazy { emptyList() }
 }
 
+@Suppress("unused")
+enum class ItemCategory(override val belongsFunc: (Item) -> Boolean) : CategorisedTargetProvider<Item> {
+    NONE({ false }),
+    TOOLS({ it is ToolItem }),
+    BLOCKS({ it is BlockItem }),
+    AXES({ it is AxeItem }),
+    SHOVELS({ it is ShovelItem }),
+    SWORDS({ it is SwordItem }),
+    PICKAXES({ it is PickaxeItem }),
+    HOES({ it is HoeItem }),
+    WEARABLE({ it is Wearable }),
```

Comment:
Carved pumpkin, dragon head, other heads (player and mobs) are not covered by `is Wearable`.